lib/gateways/plan-gateway.js

```diff
@@ -8,9 +8,10 @@ import {
 } from './models';
 
 class PlanGateway {
-  constructor({ client, tableName }) {
+  constructor({ client, tableName, logger }) {
     this.client = client;
     this.tableName = tableName;
+    this.logger = logger;
```

Comment:
We're adding a logger but not using it?

Reply:
Good point. It was used in a method that I ended up extracting. Will fix